app/lib/supabase/supabase-realtime-manager.ts

```diff
@@ -436,6 +451,19 @@ export class SupabaseRealtimeManager {
     }
   }
 
+  public simulateChannelStatusChange(
```

Comment:
I also added this so we can simulate errors/timeouts from the supabase to test our retry logic. it can be done by opneing console and running `agicashRealtime.simulateChannelStatusChange(...)`